### PR TITLE
refactor(presence): PresenceDropdown -> PresenceCollaboratorsList

### PR DESCRIPTION
**Pull request by ConradJChan**

### CODE REVIEW

File: src/features/presence/Presence.scss [hunk 1]
```diff
@@ -57,33 +57,33 @@
     }
 }
 
-.presence-dropdown {
+.bdl-PresenceCollaboratorsList {
```
Comment (jstoffan) [inline src/features/presence/Presence.scss:60]: Should these move to their own stylesheet?

File: src/features/presence/PresenceLink.js [hunk 2]
```diff
@@ -39,11 +39,7 @@ class PresenceLink extends Component {
                     <PlainButton>{children}</PlainButton>
                 </div>
                 <Overlay shouldDefaultFocus={false}>
-                    <PresenceDropdown
-                        className="presence-dropdown"
```
Comment (jstoffan) [inline src/features/presence/PresenceLink.js:43]: Is this referenced in the consuming application(s) at all?

File: src/features/presence/__tests__/__snapshots__/PresenceLink.test.js.snap [hunk 3]
```diff
@@ -25,16 +25,15 @@ exports[`features/presence/PresenceLink render() should change the flyoutPositio
...
+    <PresenceCollaboratorsList
       collaborators={
         Array [
           Object {
             "avatarUrl": "",
             "id": "1",
             "interactedAt": 999,
             "interactionType": "user.item_preview",
-            "isActive": true,
+            "isActive": false,
```
Comment (jstoffan) [inline src/features/presence/__tests__/__snapshots__/PresenceLink.test.js.snap:36]: Is this change expected?

**Review by jstoffan — DISMISSED** [review]

File: src/features/presence/__tests__/__snapshots__/PresenceLink.test.js.snap (hunk 3 above)
Comment (ConradJChan) [reply src/features/presence/__tests__/__snapshots__/PresenceLink.test.js.snap:36]: Yeah, I refactored to reuse the `collaboratorsList` mock

File: src/features/presence/PresenceLink.js (hunk 2 above)
Comment (ConradJChan) [reply src/features/presence/PresenceLink.js:43]: Not that I've seen

**Review by jstoffan — APPROVED** [review]

**Review by alexkrolick — APPROVED** [review]

**Pull request merged**